src/pages/SignUp/SignUp.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import './SignUp.scss';
+
+const SignUp = () => {
+  return (
+    <div>
```

Comment:
최상위 클래스네임 컴포넌트 이름이랑 동일하게 선언해주는 컨벤션 맞춰주세요.
```suggestion
    <div className="signUp">
```